packages/strapi-design-system/src/v2/Breadcrumbs/Breadcrumbs.js

```diff
@@ -38,11 +35,9 @@ export const Breadcrumbs = ({ label, children, ...props }) => {
   );
 };
 
-const crumbType = PropTypes.shape({ type: PropTypes.oneOf([Crumb, CrumbLink, CrumbSimpleMenu]) });
-
 Breadcrumbs.displayName = 'Breadcrumbs';
 
 Breadcrumbs.propTypes = {
-  children: PropTypes.oneOfType([PropTypes.arrayOf(crumbType), crumbType]).isRequired,
+  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]).isRequired,
```

Comment:
I'm not a fan of removing component type check but this is preventing us to create a CrumbSimpleMenuAsync component and passing it to Breadcrumbs without error, I'm open to other ideas 😁